cmd/infracost/comment_github.go

```diff
@@ -134,7 +134,7 @@ func commentGitHubCmd(ctx *config.RunContext) *cobra.Command {
 					return err
 				}
 
-				if res.Posted && ctx.IsCloudUploadExplicitlyEnabled() {
+				if res.Posted && ctx.IsCloudUploadEnabled() {
```

Comment:
This was me that did this so apoligies - I can see the code difference between thes, but what is the realterms difference in usage?